Sources/Core/Request/Request.swift

```diff
@@ -160,7 +160,11 @@ public extension Request {
 public struct RequestDefaults {
     
     public static var defaultCachePolicy: URLRequest.CachePolicy = .useProtocolCachePolicy
-    public static var defaultDecoder: JSONDecoder = JSONDecoder()
+    public static var defaultDecoder: JSONDecoder = {
```

Comment:
This is probably not something we want to do out from under people, even in a major version change. Could we instead set up `JSONDecoder.iso8601` ?

Reply:
Sure, I can change this around some. To be clear, are you talking about changing this line to

`public static var defaultDecoder: JSONDecoder = JSONDecoder.iso8601`

along with an extension on JSONDecoder to include an iso8601 static property?

Reply:
Should mention that the reason I made this change to begin with was to conform to the unit tests. And in my own usage I've only ever dealt with ISO8601 dates, so I've always been forced to provide my own JSONDecoders.

Reply:
I'd like to keep the default decoder the same as is-`JSONDecoder()`- because it's not an obvious change for anyone currently using the default decoder in their request definitions, things would just start to fail to decode if for example they were using a non-ISO date format. If we have need for an additional ISO8601 decoder, I'd suggest we add an additional property for it (whether as extension to `RequestDefaults` or `JSONDecoder` itself).

Reply:
I do understand what you're saying, and it would be frustrating to try to understand why dates are suddenly failing to decode properly. But I do believe that having `iso8601` would be a more sensible default. Not really sure what a perfect solution would be moving forward (a new initializer on `Request`, perhaps?), but for now I've just reverted the default back to int-based dates and added in a static property on `JSONDecoder`

Reply:
As long as it's heavily documented in the release notes / migration guide, then I'm OK changing it to be the new default.

Reply:
Oh man, I really don't mean to be pushing back on you too much on this. Just putting my thoughts out there.

Happy to document and change the default or leave it as is.

Reply:
I'll just leave it as is. Adding a public extension to `JSONDecoder` seems to keep the callsite really clean when implementations do need to use `iso8601` decoding.